From 5a5b343c80239aaf75d2a582920b75c8c8058da2 Mon Sep 17 00:00:00 2001
From: Christian Tismer <tismer@stackless.com>
Date: Thu, 24 Nov 2022 10:24:58 +0100
Subject: [PATCH] Fix a cmake-only build

pyminver was computed by cmake and evaluated by parser.py .
A recent refactoring extracted a computed string in a
wrong way.

Change-Id: Ia8264294ad0e050863ea912a9fee15792bed8f10
Pick-to: 6.4
Fixes: PYSIDE-2127
Fixes: PYSIDE-2128
---
 .../files.dir/shibokensupport/signature/parser.py               | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py b/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py
index be82a4f..66d79ad 100644
--- a/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py
+++ b/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py
@@ -69,7 +69,7 @@ def _get_flag_enum_option():
         flag = getattr(sys, sysname)
         if not isinstance(flag, int):
             flag = True
-    p = f"\n    *** Python is at version {'.'.join(map(str, pyminver))} now."
+    p = f"\n    *** Python is at version {'.'.join(map(str, pyminver or 0))} now."
     # PYSIDE-1797: Emit a warning when we may remove pep384_issue33738.cpp
     if pyminver and pyminver >= (3, 8):
         warnings.warn(f"{p} The file pep384_issue33738.cpp should be removed ASAP! ***")
-- 
2.38.1

